let aa = ['哈哈哈','嘿嘿嘿']
try {
    console.log('111:',aa[2].test)
} catch (error) {
    console.log('9898错误了')
}
console.log('执行到我啦')